### PR TITLE
feat: fluid spacing

### PR DESCRIPTION
**Pull request by gavmck**
Adds fluid spacing

16 -> 64px
16 -> 48px
16 -> 80px

### CODE REVIEW

File: components/canvas/Card/Card.css [hunk 1]
```diff
@@ -16,6 +16,7 @@ diamond-card {
 
   &[radius] {
     border-radius: var(--diamond-radius);
+    overflow: hidden;
```
Comment (DanWebb) [inline components/canvas/Card/Card.css:19]: Why?

File: styles/tokens/spacing.css [hunk 2]
```diff
@@ -6,4 +6,8 @@
   --diamond-spacing-sm: calc(var(--diamond-spacing) / 2);
   --diamond-spacing-lg: calc(var(--diamond-spacing) * 2);
   --diamond-spacing-xl: calc(var(--diamond-spacing) * 4);
+
+  --diamond-spacing-fluid: clamp(1rem, 5vw, 4rem);
+  --diamond-spacing-fluid-sm: clamp(1rem, 5vw, 3rem);
+  --diamond-spacing-fluid-lg: clamp(1rem, 5vw, 5rem);
```
Comment (DanWebb) [inline styles/tokens/spacing.css:12]: Should section support these?

**Review by DanWebb — APPROVED** [review]

File: components/canvas/Card/Card.css (hunk 1 above)
Comment (gavmck) [reply components/canvas/Card/Card.css:19]: Otherwise the content doesn't get clipped

File: styles/tokens/spacing.css (hunk 2 above)
Comment (gavmck) [reply styles/tokens/spacing.css:12]: Yeah, probably card too looking at EtchOS

**Pull request merged**